README.md

```diff
@@ -14,7 +14,7 @@ Javascript library to integrate [Application Insights][appinsights-js] in applic
 Using npm:
 
 ```bash
-npm install react-appinsights
+npm install --save react-appinsights@beta
```

Comment:
(strictly speaking, the save flag has been removed in the last versions of npm: by default it saves dependencies in package.json - see https://docs.npmjs.com/cli/install )